app/lib/LevelLoader.coffee

```diff
@@ -261,6 +261,9 @@ module.exports = class LevelLoader extends CocoClass
       session.patch = session.save = session.put = -> console.error "Not saving session, since we didn't create it."
     else if codeLanguage = utils.getQueryVariable 'codeLanguage'
       session.set 'codeLanguage', codeLanguage
+    if me.id is session.get('creator')
+      # do not use .set since we won't update fullName to server
+      session.fullName = _.filter([me.get('firstName'), me.get('lastName')]).join(' ')
```

Comment:
can't we use user.broadName ?

Reply:
🤔 actually we do part of `broadName` logic when displaying this fullName. And yes we can turn to broadName I believe.

Reply:
It does things on top of your logic so it might make more consistent but anyway you have merged now 😅

Reply:
ya 😂  i can update later if we find broadName is better. besides this pr there is another place we use simple `first last . join`